thentos-core/thentos-core.cabal

```diff
@@ -43,7 +43,7 @@ library
   hs-source-dirs:
       src
   ghc-options:
-      -Wall
+      -Wall -j5
```

Comment:
I think this should be left for the user to decide at the `cabal configure` stage.